From 12e220c5241128ad62caebfaad9da5a33b453441 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 5 Jun 2013 09:37:27 +0800
Subject: [PATCH 205/310] ENGR00264683 Wireless display: Cannot mirror the
 content on the TV

Add r_submix module descriptor for WFD in SabreAI

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 sabreauto_6q/audio_policy.conf | 20 +++++++++++++++++++-
 1 file changed, 19 insertions(+), 1 deletion(-)

diff --git a/sabreauto_6q/audio_policy.conf b/sabreauto_6q/audio_policy.conf
index 24bce66..b2d8511 100644
--- a/sabreauto_6q/audio_policy.conf
+++ b/sabreauto_6q/audio_policy.conf
@@ -53,7 +53,7 @@ audio_hw_modules {
       }
     }
   }
- usb {
+  usb {
     outputs {
       usb_accessory {
         sampling_rates 44100
@@ -69,4 +69,22 @@ audio_hw_modules {
       }
     }
   }
+  r_submix {
+    outputs {
+      submix {
+        sampling_rates 44100|48000
+        channel_masks AUDIO_CHANNEL_OUT_STEREO
+        formats AUDIO_FORMAT_PCM_16_BIT
+        devices AUDIO_DEVICE_OUT_REMOTE_SUBMIX
+      }
+    }
+    inputs {
+      submix {
+        sampling_rates 44100|48000
+        channel_masks AUDIO_CHANNEL_IN_STEREO
+        formats AUDIO_FORMAT_PCM_16_BIT
+        devices AUDIO_DEVICE_IN_REMOTE_SUBMIX
+      }
+    }
+  }
 }
-- 
1.8.0

